.github/workflows/ci.yml

```
@@ -0,0 +1,133 @@
+  build:
+    needs: [lint, semgrep, test, security_scan]
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write 
+      
+    steps:
+      - uses: actions/checkout@v3
+      
+      - name: Log in to the Container registry
+        uses: docker/login-action@v2
```

actionlint (1.7.10)

[error] the runner of "docker/login-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)